devel/external/AEGIS/AEGIS_25.2_training_sample_readme.txt

```diff
@@ -0,0 +1,109 @@
+This README file describes the AEGIS F814W<25.2 training sample, which is a
+randomly ordered sample of galaxies in the AEGIS field with AB magnitude<25.2, in F606W(V) and F814W(I) bands.
+
+NOTE: The columns in the catalog are those expected by galsim.RealGalaxyCatalog.
+However, some parts of that analysis, i.e parametric fits, were not performed.
+Hence those values were manually set and do not have physical meaning.
```

Comment:
OK - so it seems like ```AEGIS_galaxy_catalog_[VI]_25.2.fits``` are complete, and it's just the other files that might not be complete? If so, I would recommend moving this comment down past (1).  Its current location implies that all the catalogs may be incomplete, but it seems from your github comment that (1) is complete.